client/my-sites/plugins/search-box-header/index.jsx

```diff
@@ -39,13 +61,16 @@ const PopularSearches = ( props ) => {
 
 			<div className="search-box-header__recommended-searches-list">
 				{ searchTerms.map( ( searchTerm, n ) => (
-					<a
-						href={ `/plugins/${ siteSlug || '' }?s=${ searchTerm }` }
+					<span
```

Comment:
Is role=link enough for screen readers? (TIL if so)

Reply:
it seems to be the case https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_link_role